cirq-core/cirq/circuits/circuit.py

```diff
@@ -883,6 +883,15 @@ def all_operations(self) -> Iterator[ops.Operation]:
         """
         return (op for moment in self for op in moment.operations)
 
+    def map_ops(
+        self: CIRCUIT_TYPE, func: Callable[['cirq.Operation'], 'cirq.OP_TREE']
+    ) -> CIRCUIT_TYPE:
```

Comment:
Should this have a docstring?  Also, should this be map_operations to avoid the abbreviation?

Reply:
Done.